Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue4597.cs

```diff
@@ -129,6 +138,9 @@ public void TestImagesDisappearCorrectly()
 					}
 				}
 			}
+#else			
+			//can't validate if images have vanished until this is resolved
+			Assert.Inconclusive(@"https://github.com/xamarin/Xamarin.Forms/issues/4731");
```

Comment:
`Assert.Inconclusive` throws exception.
This test may pass only on android and ios?

Reply:
Good point. I marked this test with the category UWPIgnore for now so when we're running the suite of UWP tests it won't run

Reply:
The manual test still runs and reaches this point, but ends without exception.